docs/src/main/sphinx/installation/rpm.rst

```diff
@@ -0,0 +1,70 @@
+===========
+RPM Package
+===========
+
+Users of RedHat Enterprise Linux 7 or 8 and CentOS 7 or 8 can use the RPM package to install SEP. Other versions or distributions are not supported.
```

Comment:
We can not add specific distro since we are not testing. So lets just say the short section here

```
You can use the RPM to install Trino on Linux distributions with RPM support.
```